### DISCUSSION TITLE
MySQL DB (RDS) 구축

### DISCUSSION THREAD

**Discussion by koreanddinghwan** [manito42/backend#81] [General]

**Comment by koreanddinghwan** [comment]
## Parameter Group

### timezone

- time_zone: Asia/Seoul

--주의--

[database timezone setting](https://repost.aws/knowledge-center/rds-change-time-zone)  

- database의 timezone은 UTC로 사용하되, application level에서 변환해서 사용하는게 좋다고합니다.  
- 현재 application level에서 변환하는 로직이 없으므로 timezone은 Asia/Seoul로 변경해서 사용하겠습니다

<br><br>

### charset

- utf8: 이모지 사용불가능
- utf8mb4: 이모지 사용가능

- character_set_client : utf8mb4
- character_set_connection : utf8mb4
- character_set_database: utf8mb4
- character_set_filesystem: utf8mb4
- character_set_results: utf8mb4
- character_set_server: utf8mb4
- collation_connection: utf8mb4_general_ci
- collation_server: utf8mb4_general_ci

**Comment by JuneParkCode** [comment]
TZ 이야기를 안했었네요. API 서버 또한 Asia/Seoul 로 사용하고 있습니다.
 현재는 시간과 관련된 정보를 조회할 일이 없는데, 추후 이런 저런 기능 들어오고, 만약 타 캠퍼스 확장을 생각하면 UTC로 맞추는게 맞나 싶기도 하고 ...

**Comment by koreanddinghwan** [comment]
> TZ 이야기를 안했었네요. API 서버 또한 Asia/Seoul 로 사용하고 있습니다.
> 
>  현재는 시간과 관련된 정보를 조회할 일이 없는데, 추후 이런 저런 기능 들어오고, 만약 타 캠퍼스 확장을 생각하면 UTC로 맞추는게 맞나 싶기도 하고 ...
> 
> 

프론트와도 상의하는게 좋을 것 같습니다


시간 변환에 대한 책임을 어디에 둘 것인가? 프론트에서하면 번들링 이슈가 있을거고(Next라고 순수 리액트보다는 적음) 
백에서하면 클라이언트의 캠퍼스 정보마다 시간정보를 세팅하는 등의 의사결정이 필요하겠네여
